src/Core/Types/PrimitiveType.cs

```diff
@@ -144,6 +144,14 @@ private static PrimitiveType Create(Domain dom, int bitSize, string name)
 			return shared;
 		}
 
+        public static bool IsWord(DataType dt)
```

Comment:
I'm unhappy that we have to call `CreateWord` in a predicate method (= method testing some property and returning true or false) but we don't really have a good `IsWord` type -- yet. Please add a `//$REFACTOR: CreateWord is inefficient`, make this method non-static and then move it to `DataType`:
```C#
public bool IsWord()
{
    if (this.BitSize == 0)
        return false;
    //$REFACTOR: CreateWord is inefficient. 
    var wordType = CreateWord(this.BitSize);
    return wordType == this;
}
```

Reply:
Done. See https://github.com/uxmal/reko/compare/bc60d5dbcdde7462d50bc841ac8673cf2dd3ebeb..dc646bb14758dfcf8acc216f13a49b57ad41d68a